json_schema/type/protocol/biomaterial_collection/dissociation_protocol.json

```
@@ -54,7 +54,63 @@
                 "type": "object",
                 "$ref": "module/process/purchased_reagents.json"
             },
-            "user_friendly": "Protocol reagents"
+            "user_friendly": "Purchased reagents"
+        },
+        "digestion_time": {
+            "description": "Time of digestion of the specimen.",
+            "type": "integer",
+            "example": "2; 5; 10",
+            "user_friendly": "Digestion time"
+        },
+        "digestion_time_unit": {
+            "description": "The unit in which the digestion time is expressed.",
+            "type": "object",
+            "$ref": "module/ontology/time_unit_ontology.json",
+            "user_friendly": "Digestion time unit"
+        },
+        "digestion_temperature": {
+            "description": "Temperature of digestion in Celsius or in general terms (i.e. cold, warm, room temperature).",
+            "anyOf": [
```

I was confused but 

```suggestion
            "oneOf": [
```

is semantically equivalent (given that the alternatives are disjunctive) and adds clarity.